DEVDOCS-4799 [revise] Stencil Docs, Common Objects `has_options` description update (#2020)
bigcommerce/dev-docs: docs/stencil-docs/reference-docs/common-objects.mdx

| date_added | Date the product was added to this BigCommerce storefront | string Ex. `"Aug 15th 2018"` |
| pre_order |Product’s availability for pre-order | boolean |
| show_cart_action | Indicates whether the product is available for purchase | boolean |

> | has_options | Indicates whether customer is required to specify options when ordering the product. A `false` variable appears When product is out of stock, including it's options.  | Boolean | 

Reviewer: ```suggestion
| has_options | Indicates whether the customer must specify options when ordering the product. A `false` variable appears when the product is out of stock, including its options.  | Boolean | 
```
